kobo/apps/stripe/views.py

```diff
@@ -35,6 +35,14 @@
 from kpi.permissions import IsAuthenticated
 
 
+def generate_return_url(product_metadata):
+    base_url = settings.KOBOFORM_URL + '/#/account/'
+    return_page = (
+        'addons' if product_metadata['product_type'] == 'addon' else 'plan'
+    )
+    return base_url + return_page
+
+
```

Comment:
One small thing: this should probably be moved to kobo/apps/stripe/utils.py, just to keep this file limited to the various Stripe `View`s.